docs/FAQs/index.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: home
+title: FAQs
+nav_order: 6
+---
+
+# Estela FAQs  
+Check the list of common questions and issues in Estela, check the topics (e.g.**[Installation]**) and choose your question.
+
+## Estela Installation
+### Q: How to start again installation process if I had mistakes in the [*resources guide*]({% link estela/installation/resources.md %}){:target="_blank"}  
```

Comment:
Please check the grammar and formulate questions (using `?`). For example, this question should be: `How can I restart the installation process if I had errors in the resource guide?`. Try to be more clear and precise about the question.